### PR TITLE
Chrony machineconfig file generation

### PR DESCRIPTION
**Pull request by schabrolles**
Following the discussion initiated here (issue #65) 

This PR provides a way to generate Chrony config file from a configuration provided by the user (in a variable file).

Files will be generated into a `machineconfig` folder and must be copied manually by the user into the `manifests` directory before generating the ignition files.

### CODE REVIEW

**Comment by schabrolles** [comment]
@christianh814, I think I applied the modification we discussed in #65 (with some adjustment) 
can you please have a look ?

**Comment by christianh814** [comment]
@schabrolles Looks good! I'll run the tests and merge it to `devel` today or tomorrow (if quay.io doesn't have problems again)

File: docs/examples/vars.yaml [hunk 1]
```diff
@@ -44,3 +44,11 @@ other:
...
     ipaddr: "192.168.7.31"
     macaddr: "52:54:00:f4:2e:2e"
+
+chronyconfig:
+  enabled: false
+  content:
+    - server: 0.centos.pool.ntp.org
+      options: iburst
+    - server: 1.centos.pool.ntp.org
+      options: iburst
```
Comment (christianh814) [inline docs/examples/vars.yaml:54]: Only feedback I have is that the `chronyconfig` should not be in the "default" examples. This config should be in the `vars/main.yaml` file as...

```
chronyconfig:
  enabled: false
```

The user then overrides this setting.

I'm not opposed to having a `docs/examples/vars-chrony.yaml` example though.

**Comment by schabrolles** [comment]
HI @christianh814
I’ve push a new commit with your suggestions

**Pull request merged**